.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
```

Comment:
* Please Pull Request should have a summary describing introduced changes in more detail.
![image](https://user-images.githubusercontent.com/74744787/211909784-8610bc95-5ef9-4912-bafc-1079f956573b.png)